webapp/src/index.tsx

```diff
@@ -268,8 +268,13 @@ export default class Plugin {
 
             switch (subCmd) {
             case 'join':
```

Comment:
Need some guidance here to limit `/call join` to only join calls and not start them.

Reply:
I think it's fine to keep the existing behaviour here. If anything I'd be making `/start` fail if the call is already ongoing.

To do that I think we need a new redux selector though, more or less a copy of [`voiceConnectedUsers`](https://github.com/mattermost/mattermost-plugin-calls/blob/f252b1e7a0216fb6f4320034cfca03c1ac47ecf7/webapp/src/selectors.ts#L12-L19) but that receives a channel id given it may not always be the current channel in this case. Then if the result has length > 0 we error out since it means the call is happening already.

Reply:
We need to pass `args.channel_id` to the selector. That should fix it.

Reply:
Done